# obstacle detection msg

string id
vision_msgs/Detection3D detection
geometry_msgs/Twist twist
